quizyapp/models.py

```diff
@@ -0,0 +1,100 @@
+from django.db import models
+from django.contrib.auth.models import User
+from hashlib import md5
+from html import unescape
+from django.db.models.fields.related import ForeignKey
+import requests
+
+class UserPoints(models.Model):
+    user = models.ForeignKey(User, on_delete=models.PROTECT) 
+    points = models.IntegerField()
+
+class Category(models.Model):
+    id = models.IntegerField(unique=True, primary_key=True)
+    name = models.TextField(unique=True)
+
+    @staticmethod
+    def init_category_list_from_api_if_none_available():
+        if not Category.objects.all().count():
```

Comment:
exist() to też opcja.

Przy dużych zbiorach `.count()` jest niewydajne.

Reply:
funkcja przeniesiona do osobnego modułu i poprawiona